packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -299,6 +303,20 @@ const ActivityDetailsSidebar = (props: Props) => {
     history.push(`/me/organizations/${selectedTeam.orgId}/billing`)
   }
 
+  const meetingNamePlaceholder =
```

Comment:
+1 I think this should just be "Enter the name of your meeting" rather than the meeting type. As user could think the placeholder is the actual value of the text, whereas "Enter the name of your meeting" makes it clearer that they need to update the input in my opinion.

Reply:
They don't need to enter anything, we will default to the placeholder. Why forcing the extra step on them?

Reply:
Ah I thought it was a required input